src/Polymesh.ts

```diff
@@ -78,6 +79,21 @@ export class Polymesh {
 
   static async connect(params: ConnectParamsBase & { accountUri: string }): Promise<Polymesh>;
 
+  static async connect(
```

Comment:
These types are slightly off. The SDK should allow connecting without any seed, keyring or URI but still connect to the middleware. These signatures don't allow that. You should put the harvester config as an optional property of `ConnectParamsBase` and remove it from these overloads